server_cmds.go

```diff
@@ -49,6 +49,21 @@ type ServerConnectionInfo struct {
 	ConnectedTime                 uint32  `ms:"connection_connected_time"`
 	PacketLossTotalAvg            float32 `ms:"connection_packetloss_total"`
 	PingTotalAvg                  float32 `ms:"connection_ping"`
+
+	SpeechPacketsSent     		  uint64  `ms:"connection_packets_sent_speech"`
+	SpeechBytesSent       		  uint64  `ms:"connection_bytes_sent_speech"`
+	SpeechPacketsReceived 		  uint64  `ms:"connection_packets_received_speech"`
+	SpeechBytesReceived   		  uint64  `ms:"connection_bytes_received_speech"`
+	
+	KeepalivePacketsSent     	  uint64  `ms:"connection_packets_sent_keepalive"`
+	KeepaliveBytesSent       	  uint64  `ms:"connection_bytes_sent_keepalive"`
+	KeepalivePacketsReceived 	  uint64  `ms:"connection_packets_received_keepalive"`
+	KeepaliveBytesReceived   	  uint64  `ms:"connection_bytes_received_keepalive"`
+
+	ControlPacketsSent     		  uint64  `ms:"connection_packets_sent_control"`
+	ControlBytesSent       		  uint64  `ms:"connection_bytes_sent_control"`
+	ControlPacketsReceived 		  uint64  `ms:"connection_packets_received_control"`
+	ControlBytesReceived   		  uint64  `ms:"connection_bytes_received_control"`
```

Comment:
Just spotted the language here is reversed, which may cause confusion even though the read better we've tried to stick to the original wording as much as possible so its easy to convert between raw and struct.

So I believe we should stick with this and rename `SpeechPacketsSent` to `PacketsSentSpeech` etc.